docs/openshift.asciidoc

```diff
@@ -29,9 +29,19 @@ Before deploying an Elasticsearch cluster with ECK, make sure you correctly appl
 [float]
 [id="{p}-openshift-deploy-the-operator"]
 === Deploy the operator
+
 . Apply the all-in-one template, as described in the link:k8s-quickstart.html[quickstart].
 +
-[source,shell,subs="attributes"]
+If you are using a Kubernetes cluster in version 1.12 or below, use:
++
+[source,sh,subs="attributes"]
+----
+oc apply -f https://download.elastic.co/downloads/eck/{eck_version}/all-in-one-no-validation.yaml
```

Comment:
It does not seem to be necessary:

```
oc version
oc v3.11.0+0cbc58b
kubernetes v1.11.0+d4cacc0
features: Basic-Auth

Server https://openshift-ansible.35.234.72.215.nip.io:8443
openshift v3.11.0+bdd37ad-314
kubernetes v1.11.0+d4cacc0
```

```
oc apply -f https://download.elastic.co/downloads/eck/1.0.0-beta1-bc10/all-in-one.yaml
customresourcedefinition.apiextensions.k8s.io/apmservers.apm.k8s.elastic.co created
customresourcedefinition.apiextensions.k8s.io/elasticsearches.elasticsearch.k8s.elastic.co created
customresourcedefinition.apiextensions.k8s.io/kibanas.kibana.k8s.elastic.co created
clusterrole.rbac.authorization.k8s.io/elastic-operator created
clusterrolebinding.rbac.authorization.k8s.io/elastic-operator created
namespace/elastic-system created
statefulset.apps/elastic-operator created
serviceaccount/elastic-operator created
```

Reply:
Ok. That's disturbing. I'm going to put this on hold until we confirm that it works also on an "Enterprise" OpenShift.